com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -306,6 +315,13 @@ inline void InitializeBRDFDataDirect(half3 diffuse, half3 specular, half reflect
 #endif
 }
 
+// Legacy: do not call, will not correctly initialize albedo property.
+inline void InitializeBRDFDataDirect(half3 diffuse, half3 specular, half reflectivity, half oneMinusReflectivity, half smoothness, inout half alpha, out BRDFData outBRDFData)
+{
+    InitializeBRDFDataDirect(half3(0.0, 0.0, 0.0), diffuse, specular, reflectivity, oneMinusReflectivity, smoothness, alpha, outBRDFData);
+}
```

Comment:
Could we move this function to `Deprecated.hlsl`?

Reply:
Actually I can't. This function relies on `struct BRDFData` to be declared, which is at the top of `Lighting.hlsl`.
However, our include order is:
ShaderLibrary\Core.hlsl -> ShaderLibrary/Deprecated.hlsl -> ShaderLibrary\Lighting.hlsl

So I cannot move `InitializeBRDFDataDirect()` in `Deprecated.hlsl`.
Unless we refactor the code and split globals/struct declarations in a different header from functions?

Reply:
Ahh I see. That's a very good idea!
I won't let you worry about that work though. I'm noting this down and will talk to the URP team about this as I've been meaning to clean up our shader code for a while and this would fit into that.